packages/@react-aria/landmark/src/useLandmark.ts

```diff
@@ -140,14 +147,21 @@ class LandmarkManager {
   private checkLabels(role: AriaLandmarkRole) {
     let landmarksWithRole = this.getLandmarksByRole(role);
     if (landmarksWithRole.size > 1) {
-      if ([...landmarksWithRole].some(landmark => !landmark.label)) {
-        console.warn(`Page contains more than one landmark with the '${role}' role. If two or more landmarks on a page share the same role, all must be labeled with an aria-label or aria-labelledby attribute.`);
+      let duplicatesWithoutLabel = [...landmarksWithRole].filter(landmark => !landmark.label);
+      if (duplicatesWithoutLabel.length > 0) {
+        console.warn(
```

Comment:
Should these be `console.warn` or `console.error`? Right now we have `console.error` for when you try to add more than one main landmark.